app/models/submission.js

```diff
@@ -16,7 +16,7 @@ export default Model.extend({
   explanation: DS.attr(),
   is_top_submission: DS.attr(),
   plagiarism_detected: DS.attr(),
-  content: DS.belongsTo('content'),
+  content: DS.belongsTo('content', { inverse: null }),
```

Comment:
inverse null is not right. since any update on content model will not reflect on submission model.
instead create hasMany relationship on content model and set inverse to that relation

Reply:
content can have many submissions